ConsumerApi.Tests.Integration/Features/Relationships/POST.feature

```diff
@@ -0,0 +1,64 @@
+﻿@Integration
+Feature: POST Relationship
+
+User creates a Relationship
+
+Scenario: Creating a Relationship
+	Given Identities i1 and i2
+	And a Relationship Template rt created by i1
+	When a POST request is sent to the /Relationships endpoint by i2 with rt.id
+	Then the response status code is 201 (Created)
+	And the response contains a Relationship
+
+Scenario: Creating a Relationship with Identity "ToBeDeleted"
```

Comment:
This sounds as if
* the identity was called "ToBeDeleted"
* you try to create the relationship **as** the identity in that status

The first bullet point also applies to the other tests.

```suggestion
Scenario: Creating a Relationship to an Identity in status "ToBeDeleted"
```